src/checkbox-group.test.basics.ts

```diff
@@ -66,6 +66,28 @@ it('does not include in its `value` disabled checkboxes that are checked', async
   expect(checkboxes[1]?.checked).to.be.true;
 });
 
+it('sets `privateVariant` on added Checkboxes', async () => {
```

Comment:
Open to suggestions on the test title here as well. It's brief enough and to the point of what we expect, but not sure if there's something better.

Reply:
Not a big deal, but Checkboxes present on initial render are also _added_:

```suggestion
it('sets `privateVariant` on Checkboxes added after initial render', async () => {
```